core-common/src/main/java/org/glassfish/jersey/logging/LoggingFeature.java

```diff
@@ -222,7 +223,6 @@ public LoggingFeature(Logger logger, Level level, Verbosity verbosity, Integer m
                 .level(level)
                 .verbosity(verbosity)
                 .maxEntitySize(maxEntitySize)
-                .separator(DEFAULT_SEPARATOR)
```

Comment:
The removal of the `DEFAULT_SPEARATOR` is fine here, but should the `configureBuilderParameters` be updated from `builder.separator = builder.separator == null ? filterSeparator : builder.separator;` to `builder.separator = builder.separator == null ? filterSeparator : DEFAULT_SEPARATOR`?

Reply:
If we look at some lines above, we can see that the default separator is already considered when the configs are read (same behavior as other configs):
https://github.com/eclipse-ee4j/jersey/blob/9bb2e8dffe2ee307dde151b35f3b98fc81782d4c/core-common/src/main/java/org/glassfish/jersey/logging/LoggingFeature.java#L289-L295

Also, notice that with such change we would always discard what was set in the `builder`.